test/node/parsing/WebGL/6

```diff
@@ -0,0 +1,552 @@
+typedef ([AllowShared] Uint32Array or sequence<GLuint>) Uint32List;
```

Comment:
The comment in `Parser.js` seems to imply that it was designed around HeyCam's Grammar. Perhaps there were changes to the spec / grammar between the time the Parser was written and the current specs. I am currently working on adding this change. Hoping to have a PR before the end of the day today (hopefully).